.github/ISSUE_TEMPLATE/1-new-plugin-request.yml

```
@@ -30,6 +30,13 @@ body:
       placeholder: "For example: Authenticate the CLI using a Personal Access Token."
     validations:
       required: true
+  - type: checkboxes
+    id: contribution
+    attributes:
+      label: Contribution
+      description: Shell plugins are a community effort, so you're more than welcome to [build the plugin yourself](https://developer.1password.com/docs/cli/shell-plugins/contribute/)!
```

Better?
```suggestion
      description: Shell plugins are a community effort, so you're more than welcome to [build the plugin yourself](https://developer.1password.com/docs/cli/shell-plugins/contribute/)! We're here to help you out if needed.
```

---

We need to be careful that this is not interpreted as a commitment that we'll be building it. You could consider a slight variation:

```suggestion
      description: Shell plugins are a community effort, so you're more than welcome to [build the plugin yourself](https://developer.1password.com/docs/cli/shell-plugins/contribute/)! We're here to help when you have any questions.
```